generator-dialtone/generators/vue2/templates/story.ejs

```diff
@@ -17,14 +17,17 @@
       We use this storybook control naming scheme to prevent conflicts between controls for props and slots with the
       same name.
     -->
-    <template v-if="defaultSlot">
-      <span v-html="defaultSlot" />
+    <template>
+      <span
+        v-if="$attrs.default"
+        v-html="$attrs.default"
+      />
     </template>
```

Comment:
We can remove this <template> tag now that we no longer need a v-if

Reply:
Right, forgot about that, done